src/Components/Icon/Icon.tsx

```diff
@@ -0,0 +1,11 @@
+import { css } from 'aphrodite';
+import styles from './styles';
+import { FaBars } from "react-icons/fa";
+
+export const Icon = () => {
+    return (
+        <FaBars className={css(styles.iconito)} />
```

Comment:
change `styles.iconito` to `styles.icon`

Reply:
Stile name changed